tools/mcsema_disass/ida7/get_cfg.py

```diff
@@ -90,6 +99,28 @@
 
 _NOT_ELF_BEGIN_EAS = (0xffffffffL, 0xffffffffffffffffL)
 
+## Added function to handle the dynsym section by elftoos,
+## which can get the external variable's size
+def extract_external_size():
+    DEBUG("Looking for the external variable and it's size")
+    global EXTERNAL_VARS_SIZE
+    if binary_path == None:
+        return
+    openFile = open(binary_path, 'rb')
```

Comment:
Can you follow the naming scheme `var_name` instead of `varName`?

Reply:
Also:
1) Check if the binary is an ELF, there is likely an API in IDA for this.
2) Move the `elffile` import into its own function, wrapping it with a `try ... except`, so that if it isn't installed, things still work. Print out an ERROR into the log when it's missing.

Reply:
Mostly though, this seems to be checking the size of the extern var. Have you checked to see if IDA Pro already produces the correct info?